src/components/Kv/KvLightbox.vue

```diff
@@ -160,6 +160,7 @@ export default {
 
 			#lightbox-columns {
 				position: relative;
+				width: 100%;
```

Comment:
ensuring this parent container is set to 100% of the parent instead of stretching to the full width of child content.

Reply:
What was the issue this fixed? Would adding a class of `small-12` have the same effect?

Reply:
I'll look into this.